src/pages/components/Sidebar/SidebarItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Box, Flex, FlexProps } from '@chakra-ui/react';
+
+interface IconType {
+  size?: string | number | boolean;
+}
+
+interface SidebarItemProps extends FlexProps {
+  icon: React.ReactElement<IconType>;
+  children: React.ReactNode;
+}
+
+const SidebarItem: React.FC<SidebarItemProps> = function SidebarItem({ icon, children }) {
+  return (
+    <Box as="a" href="#" style={{ textDecoration: 'none' }} _focus={{ boxShadow: 'none' }}>
```

Comment:
should `href` have a dynamic value that comes from props?

Reply:
Yeah, just added